stockAndPrices/services/app/main.py

```diff
@@ -64,17 +65,19 @@ def dblogin():
     except:
         rtn = "Connection Error! Please provide valid dbHost and dbPort as query strings and make sure your" \
               " mongoDB server is running."
-        return f"<h1>{rtn}<h1>"
+        return f"<h1>{rtn}<h1>", 500
 
     sample_db = client.sampleDB
     dev_col = sample_db.developer
     if request.method == "GET":
         if dev_col.find_one({"first": "Hello"}) is None:
-            rtn = ": No data with Hello World in db."
+            rtn = "No data with Hello World in db."
+            http_code = 209
         else:
             document = dev_col.find_one({"first": "Hello"})
             rtn = document["first"] + " " + document["last"]
-        return f"<h1>Login Page {rtn}<h1>"
+            http_code = 200
```

Comment:
The http_code is initialized to 200 already, this does not need to be set to 200 again.